postgresql-async/src/test/java/com/github/aysnc/sql/db/postgresql/util/URLParserSpec.kt

```diff
@@ -86,6 +86,19 @@ class URLParserSpec {
         assertThat(configuration.ssl.mode).isEqualTo(SSLConfiguration.Mode.VerifyFull)
     }
 
+    @Test
+    fun `postgresql URLParser should  create a connection , schema defined`() {
```

Comment:
can you please also add a test that uses the schema (select on a table without qualifying the table by schema)?

Reply:
I'll have a go later today.